includes/admin/settings/display-settings.php

```diff
@@ -98,7 +98,8 @@ function edd_admin_header() {
 			<?php if ( ! empty( $page_title ) ) : ?>
 			<span class="edd-header-page-title-wrap">
 				<span class="edd-header-separator">/</span>
-				<h1 class="edd-header-page-title"><?php echo esc_html( $page_title ); ?></h1>
+				<?php $element = true === $is_single_view ? 'span' : 'h1'; ?>
+				<<?php echo $element; ?> class="edd-header-page-title"><?php echo esc_html( $page_title ); ?></<?php echo $element; ?>>
```

Comment:
I know this isn't modifiable, as it's defined just before, but should it be echoed through `esc_attr`?

Reply:
@robincornett I don't think there's any real risk of not escaping it, but sure why not :) just to be safe.